waku/waku_metadata/protocol.nim

```diff
@@ -91,11 +93,27 @@ proc initProtocolHandler*(m: WakuMetadata) =
   m.handler = handle
   m.codec = WakuMetadataCodec
 
+proc clusterAndShards(enr: Record): Option[(uint32, HashSet[uint32])] =
```

Comment:
Does this belong in the waku_enr module?

Reply:
Well no, maybe a rename? I needed a proc to either give the cluster and shards in the ENR OR just the cluster from the config.